topbar.js

```diff
@@ -428,7 +428,7 @@ function enable () {
 
         imports.mainloop.timeout_add(0, () => {
             for (let [workspace, space] of Tiling.spaces) {
-                space.label.set_position(Math.round(r.x), Math.round(r.y));
+                space.label.set_position(Main.panel.actor.x + Math.round(r.x), Main.panel.actor.y + Math.round(r.y));
```

Comment:
Hmm, this will probably go wrong when the panel is on a monitor where `monitor.x != 0`.

Reply:
yeah, I realized that after merging, but it seems the actor is inside the panelBox so it shouldn't be a problem.

Reply:
That also means it's still not 100% robust ofc. But maybe we're lucky and themes can't change the panelbox.